gatsby/howitworks.rst

```diff
@@ -15,26 +15,31 @@ This is done with `source plugins`. The active Gatsby community has `several plu
 2) Build
 --------
 In this step there are three main technologies involved: GraphQL, React and Webpack.
+
 GraphQL is a query language used to create apis where you can create queries that generates exactly the data that you need (and only the data that you need).
+
 It is used to store and serve fetched data in a common and easily to access way.
 
-The second component is ReactJS.
-React is a javascript library that allows to create powerful and reusable interfaces.
+The second component is ReactJS: a javascript library that allows to create powerful and reusable interfaces.
```

Comment:
JavaScript per https://docs.plone.org/about/contributing/word_choice.html#preferred-word-choice

"allows one to"